.github/workflows/comment-on-e2e-check.yaml

```
@@ -0,0 +1,108 @@
+      - name: Extract data
+        id: artifact-data
+        run: |
+          echo "pr_number=$(head -n 1 pr_number.txt)" >> $GITHUB_OUTPUT
+          echo "pr_author=$(head -n 1 pr_author.txt)" >> $GITHUB_OUTPUT
+          echo "check_result=$(cat check_result.json)" >> $GITHUB_OUTPUT
+
+  comment-on-pr:
+    needs:
+      - download-artifact-data
+    runs-on: ubuntu-latest
+    permissions: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)